[2024-05-30T12:05:56.240+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-30T12:05:56.265+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stored_procedures.bash_task_end manual__2024-05-30T12:05:31.412772+00:00 [queued]>
[2024-05-30T12:05:56.272+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stored_procedures.bash_task_end manual__2024-05-30T12:05:31.412772+00:00 [queued]>
[2024-05-30T12:05:56.274+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-30T12:05:56.287+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): bash_task_end> on 2024-05-30 12:05:31.412772+00:00
[2024-05-30T12:05:56.295+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=931) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-30T12:05:56.297+0000] {standard_task_runner.py:63} INFO - Started process 933 to run task
[2024-05-30T12:05:56.296+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'stored_procedures', 'bash_task_end', 'manual__2024-05-30T12:05:31.412772+00:00', '--job-id', '329', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmphvj3r97a']
[2024-05-30T12:05:56.299+0000] {standard_task_runner.py:91} INFO - Job 329: Subtask bash_task_end
[2024-05-30T12:05:56.349+0000] {task_command.py:426} INFO - Running <TaskInstance: stored_procedures.bash_task_end manual__2024-05-30T12:05:31.412772+00:00 [running]> on host 29c09f03cc50
[2024-05-30T12:05:56.431+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Yamin' AIRFLOW_CTX_DAG_ID='stored_procedures' AIRFLOW_CTX_TASK_ID='bash_task_end' AIRFLOW_CTX_EXECUTION_DATE='2024-05-30T12:05:31.412772+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-30T12:05:31.412772+00:00'
[2024-05-30T12:05:56.433+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-30T12:05:56.445+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-30T12:05:56.446+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Here is your bash message at the end of your DAG"']
[2024-05-30T12:05:56.458+0000] {subprocess.py:86} INFO - Output:
[2024-05-30T12:05:56.460+0000] {subprocess.py:93} INFO - Here is your bash message at the end of your DAG
[2024-05-30T12:05:56.460+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-30T12:05:56.461+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-30T12:05:56.485+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=stored_procedures, task_id=bash_task_end, execution_date=20240530T120531, start_date=20240530T120556, end_date=20240530T120556
[2024-05-30T12:05:56.512+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-30T12:05:56.531+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-30T12:05:56.534+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
